docs/data/toolpad/getting-started/overview.md

```diff
@@ -0,0 +1,34 @@
+# Overview
+
+<p class="description">
+Learn core concepts of the Toolpad app and you are ready to ship your application in minutes!
+</p>
+
+## What is Toolpad?
+
+Toolpad is a self-hosted low-code internal tools builder designed to extend the MUI React components. It's designed for developers of all trades who want to save time building internal applications. Drag & drop pre-built UI components, connect your data sources, release your app - and you're done!
+
+![Editor screenshot](/static/toolpad/marketing/index-hero-video-poster.jpg)
+
+## How Toolpad works?
+
+- [Build UI](/toolpad/core-concepts/building-ui/) using our drag & drop editor
+- [Configure and fetch](/toolpad/core-concepts/connecting-to-datasources/) data from HTTP endpoints, Google sheets or even directly from PostgreSQL
+- Connect data with UI elements using [binding expressions](/toolpad/core-concepts/data-binding/)
+- When you are ready to share - [deploy](/toolpad/core-concepts/versioning-and-deploying/) a new version
```

Comment:
Rewriting this section to match the other Overview pages. Also removing all of the links here as we don't want to overwhelm the reader and send them in too many different directions at once.

```suggestion
- **Build faster than ever before:** measure your development time in minutes rather than hours or days—simply drag and drop the components you need.
- **Extensible with code:** build low-code with pro-code extensibility. You can write JavaScript anywhere in Toolpad as soon as you feel limited by the built-in features.
- **Connect to any data source:** Toolpad lets you work with HTTP endpoints, Google Sheets, or PostgreSQL for backend functionality.
- **Robust versioning and deployment:** use a Git workflow to seamlessly manage deployment and versioning of your app.
```